styles.css

```diff
@@ -45,12 +45,54 @@ a:active {
   color: limegreen;
 }
 
-/* lets connect links*/
+/*   /////CONNECT LINKS/////   */
 .connect {
   list-style-type: none;
   font-weight: bold;
 }
 
+/*   /////FORM STYLING/////   */
+input {
```

Comment:
Shared rules like
```
  border: 1px solid limegreen;
  background-color: #303030;
```
could be factored out: If you decide to change an element of the style, you only want to make that change in one place. (Down the road, syntaxes like less/sass can help with this too.)